demisto_sdk/runners/playbook_runner.py

```diff
@@ -0,0 +1,123 @@
+import os
+import time
+import demisto_client
+from demisto_client.demisto_api.rest import ApiException
+from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS
+
+
+class PlaybookRunner:
+    """PlaybookRunner is a class that's designed to run a playbook in a given Demisto instance.
+
+    Attributes:
+        playbook_id (str): the ID of the playbook to run
+        should_wait (bool): whether to wait until the playbook run is completed or not.
+        timeout (int): timeout for the command. The playbook will continue to run in Demisto
+        base_link_to_workplan (str): the base link to see the full playbook run in Demisto.
+        demisto_client (demisto_client): object for creating an incident in Demisto.
+    """
+    DEMISTO_API_KEY_ENV = 'DEMISTO_API_KEY'
+
+    def __init__(self, playbook_id: str, url: str, wait: bool, timeout: int):
+        self.playbook_id = playbook_id
+        self.should_wait = wait
+        self.timeout = timeout
+        self.base_link_to_workplan = f'{url}/#/WorkPlan/'
+        self.demisto_client = demisto_client.configure(
+            base_url=url,
+            api_key=self.get_api_key(),
+            verify_ssl=False)
+
+    def run_playbook(self):
+        # type: () -> int
+        """Run a playbook in Demisto.
+
+        Returns:
+            int. 0 in success, 1 in a failure.
+        """
+        # create an incident with the given playbook
+        incident_id = self.create_incident_with_playbook(
+            incident_name=f'inc_{self.playbook_id}', playbook_id=self.playbook_id)
+        if incident_id == -1:
+            return 1
+
+        work_plan_link = self.base_link_to_workplan + str(incident_id)
+        if self.should_wait is True:
```

Comment:
How was this comment resolved? strange
It works as I suggested earlier, added tests as example
<img width="346" alt="Screen Shot 2019-12-29 at 9 08 13" src="https://user-images.githubusercontent.com/33782301/71553591-da683480-2a1a-11ea-883a-cbf7820d8345.png">
<img width="346" alt="Screen Shot 2019-12-29 at 9 08 29" src="https://user-images.githubusercontent.com/33782301/71553592-db00cb00-2a1a-11ea-996c-e16f92860988.png">

Reply:
Done

Reply:
The problem was that click sends this argument as a string and not a bool,
therefore converted to bool in the constructor